dev-utils/storybook/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "@polypoly-eu/storybook",
+  "private": "true",
+  "scripts": {
+    "storybook:lit": "web-dev-server -c .storybook/lit/server.mjs",
+    "storybook:react": "start-storybook -p 6006 -c .storybook/react"
```

Comment:
So react and lit elements use a totally different script to run?

Reply:
I was asking more about the fact that you are using web-dev-server in one case and start-storybook in the other...

Reply:
I've tried to use the same setup for React, but the dev-server-storybook plugin only supports preact and web-components. An alternative would be to give it up and use only storybook, but then we loose rollup, since the default Storybook only uses webpack....

Reply:
No problem. Thanks for the clarification. Closing then.